package.json

```diff
@@ -2,12 +2,33 @@
   "name": "app",
   "private": "true",
   "dependencies": {
-    "@hotwired/stimulus": "^3.2.1",
-    "@hotwired/turbo-rails": "^7.3.0",
-    "webpack": "^5.88.1",
-    "webpack-cli": "^5.1.4"
+    "@reduxjs/toolkit": "^1.9.5",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0",
+    "react-redux": "^8.1.1",
+    "react-router-dom": "^6.14.1",
+    "webpack": "^5.88.1"
   },
   "scripts": {
-    "build": "webpack --config webpack.config.js"
+    "build": "esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds --loader:.js=jsx"
```

Comment:
- [ ] I get the below error when I try to start the app. This error is from the 'build script' in your package.json file. I could not test your project to see how it works. I noticed that you have set up the project code very well. 

<img width="558" alt="image" src="https://github.com/ngeelani48/hello-rails-react/assets/84156714/70d56851-8094-4ce0-b654-da1caad813f2">

Try to fix the issue with starting up your project using the commands in your package.json script and the script in `./bin/dev` as currently I get the above error.

Reply:
@bobb-Rob Thanks again 

But I have tried many things with this command still unable to resolve the issue:

![Screenshot (107)](https://github.com/ngeelani48/hello-rails-react/assets/117263981/c597ef32-fc6c-40bd-b091-67077a4c2e44)

If you @bobb-Rob can guide me about this issue I am having?

Reply:
Fixed.